qa/common/gen_qa_models.py

```diff
@@ -1212,13 +1272,29 @@ def create_plan_modelconfig(
             model_name,
             max_batch,
             version_policy_str,
-            np_to_model_dtype(input_dtype),
+            (
+                trt_dtype_to_model_dtype(input_dtype)
```

Comment:
Move this special handling within np_to_model_dtype? Otherwise it is a lot of duplicate code.